packages/core-data/src/actions.js

```diff
@@ -136,11 +136,8 @@ export function receiveEmbedPreview( url, preview ) {
  * @return {Object} Action object.
  */
 export function* editEntityRecord( kind, name, recordId, edits, options = {} ) {
-	const { transientEdits = {}, mergedEdits = {} } = yield select(
```

Comment:
I wonder if we should just ignore the edits if done early. The reason being that a config that comes later can change what these edits should really do.

Reply:
Yeah that could get very confusing.

But, ignoring the edits is just as confusing.

I think that we should throw from `editEntityRecord`, and return an empty object from `getEntityRecordNonTransientEdits` in these cases. What do you think?

Reply:
Works for me